notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -5765,6 +5765,51 @@ coping with saying goodbye forever [Gary and Mary in the end]
 bird in a gilded cage [Gary was under house arrest in a fabulous villa]
 
 
+movie: Resident Evil: Extinction (2007)
+=======================================
+
+:: Title
+Resident Evil: Extinction
+
+:: Description
+Resident Evil: Extinction is a 2007 science fiction action horror film 
+directed by Russell Mulcahy and written by Paul W. S. Anderson. A direct
+sequel to Resident Evil: Apocalypse (2004), it is the third installment in the
+Resident Evil film series, which is loosely based on the video game series of
+the same name.
+
+Synopsis: A convoy of survivors attempt to travel across the Mojave desert
+wilderness to Alaska and escape a zombie apocalypse.
+
+:: Date
+2007-09-20
+
+:: References
+https://en.wikipedia.org/wiki/Resident_Evil:_Extinction
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+zombie [A main novelty of the film is that most of the world's population has been transformed into mindless, flesh-eating zombies by an out of control virus.]
+human vs. zombie [The story follows a small group of survivors as they seek refuge from a zombie apocalypse.]
+monster apocalypse [The story is set in the immediate aftermath of a zombie apocalypse and follows a caravan of survivors who are seeking to make their way to a safe haven.]
+post-apocalyptic dystopia [The story is set in the immediate aftermath of a virus having proliferated all over the world, leaving not only the entire human and animal population turned to zombies, but also the global environment ravaged. In this desolate future, a group of survivors were trying to stay alive as best they could.]
+
+:: Major Themes
+human cloning [A main novelty of the film is that the Umbrella Corporation was producing numerous Alice clones.]
+speculative ability [The heroine of the story had heightened speed, strength, and agility. These abilities she used to kick the crap out of zombies.]
+fabled land of goodness [A convoy of survivors of the zombie apocalypse was heading to Alaska - a place that was thought to be the last refuge on Earth.]
+bioengineered human pathogen [A genetically engineered virus, known as the T-virus, had turned most of the world's human population into zombies.]
+speculative genetic engineering [A genetically engineered virus, known as the T-virus, had turned most of the world's human population into zombies.]
+
+:: Minor Themes
+telekinetic ability [Alice levitated a stone using the power of her mind to her surprise. She later used her newfound telekinetic powers to defend a convoy of survivors from a murder of zombified crows.]
```

Comment:
Imho became a major theme on account of every part of the story building up to her showing off some magic

Reply:
Hum... you speak as if you've seen this crappy movie before. In any case, there is no objection. I knocked it up and reworked the motivation accordingly.